config/config.go

```diff
@@ -69,5 +70,8 @@ func NewConfig() *Config {
 		TmetricAPIV3BaseUrl:                "https://app.tmetric.com/api/v3/",
 		TmetricDummyProjectId:              tmetricDummyProjectId,
 		TmetricTagTransferredToOpenProject: "transferred-to-openproject",
+		TmetricExternalTaskLink:            "https://community.openproject.org/",
+		// this value has always to be "https://community.openproject.org"
+		// otherwise tmetric does not recognize the integration and does not allow to create the external task
```

Comment:
Can we move the comment just above the line the comment is for?
```suggestion
		// this value has always to be "https://community.openproject.org"
		// otherwise tmetric does not recognize the integration and does not allow to create the external task
		TmetricExternalTaskLink:            "https://community.openproject.org/",
```